src/meson.build

```diff
@@ -0,0 +1,59 @@
+# We create symlinks to the native extensions in the source tree for the tests
+# and for development. Meson does not allow you to build in-place and Python
+# cannot create a single namespace from two identically-named paths.
+
+c_args = [
+    '-D_FILE_OFFSET_BITS=64',
```

Comment:
This is actually a bit funny, because meson gives you this  for free: https://github.com/mesonbuild/meson/blob/c2ed846b64f02653d1460855c0aadef20308fc1f/mesonbuild/compilers/compilers.py#L1131-L1148

https://github.com/mesonbuild/meson/commit/853634a48da025c59eef70161dba0d150833f60d


Doing anything is probably an ABI break, in the meantime meson sort of forces you into it.

Reply:
Yeah, we could drop `-D_FILE_OFFSET_BITS` given that meson already enables it.

We never actually needed `_LARGEFILE_SOURCE` or `_LARGEFILE64_SOURCE`, so those can be dropped as well.

Reply:
I knew `-D_FILE_OFFSET_BITS=64` was already included, at least in recent versions, but I thought I'd include it with the others for the avoidance of doubt. If we don't even need the others though then I guess I'll just drop them all.